src/components/PopoverContent/PopoverContent.module.css

```diff
@@ -0,0 +1,138 @@
+/*------------------------------------*\
+    # POPOVER CONTENT
+\*------------------------------------*/
+
+/**
+ * A floating panel component with arbitrary content.
+ * Styling for the Popover component since the parent <Popover> component provides functionality.
+ */
+.popover-content {
+  z-index: var(--eds-z-index-top);
```

Comment:
Is 1060 in EDSCandidates due to some bootstrap conventions but placing on top makes sense